### ISSUE TITLE
Many elements are not accessible by using Screen reader or VoiceOver

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#888]
#### Describe the bug

Many elements in SD Connect are not accessible by ScreenReader or VoiceOver. One of reasons is its lacking of `aria-*` attributes in several components.

#### To Reproduce
Steps to reproduce the behaviour:

#### Expected behavior

Elements/components in SD Connect have `aria-*` attributes attached to them.
